### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by HKalininQA**
Done homework 5

### CODE REVIEW

File: hw5/hw5_3.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+with open(file_path, 'r') as text:
+    for line in text:
+        for let in line:
+            if let.isalpha():
+                let = let.lower()
+                if let in letter_count:
+                    letter_count[let] += 1
+                else:
+                    letter_count[let] = 1
```
Comment (bohdandovbsyh) [inline hw5/hw5_3.py]: the algorithm is TOO complicated. Please find more elegant solution without nested loops

File: hw5/hw5_3.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+for letter, count in letter_count.items():
+    print(f"{letter}: {count}")
+
+
```
Comment (bohdandovbsyh) [inline hw5/hw5_3.py]: You dop not need so many empty lines at the end of file

File: hw5/hw5_3.py (hunk 1 above)
Comment (HKalininQA) [reply hw5/hw5_3.py]: I reworked solution, now the text from text.txt goes into a string and conver into a lower case, then I just iterating thorugh the string to sort the letters.

File: hw5/hw5_3.py (hunk 2 above)
Comment (HKalininQA) [reply hw5/hw5_3.py]: Lines were deleted.

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**